scripts/provision/configure_environment_recurring.sh

```diff
@@ -25,4 +39,26 @@ fi
 # Upgrade existing environment
 if [ -f ${vagrant_dir}/.idea/deployment.xml ]; then
     sed -i.back "s|magento2ce/var/generation|magento2ce/var|g" "${vagrant_dir}/.idea/deployment.xml"
-fi
\ No newline at end of file
+fi
+
+# Setup PHP
+sudo add-apt-repository ppa:ondrej/php
+sudo apt-get update
```

Comment:
We must not invoke `apt-get update` on every `vagrant up` since it will slow down the booting process significantly. We need to invoke this only if needed.